### PR TITLE
Clearify beginner wording

### PR DESCRIPTION
**Pull request by bitboxer**
When setup up the app this morning I had a little problem understanding a small part of the readme because of layout issues. I fixed those. And I removed a hard to understand word 👍

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -115,19 +115,26 @@ directory the variables from `.env` will be loaded into the environment.
...

+    * In Rails Console:   
+      ```
+      user = User.last
+      user.roles.create(name: "organizer")
+      ```
+      Instead of using the last created User, you can also search for a
+      specific User with ```user = User.find_by(github_handle: "yourgithubhandle")```.
+
+      You can assign yourself other roles in the same way. If however you assign
```
Comment (F3PiX) [inline README.md]: 'the same way' is not so clear, with the sequence changed. Maybe something as : the same way as creating the organizer role.

**Comment by F3PiX** [comment]
Nice!!!

File: README.md (hunk 1 above)
Comment (bitboxer) [inline README.md]: fixed

**Comment by carpodaster** [comment]
Merci! Documentation feedback and fixes are so incredibly valuable!

**Comment by alicetragedy** [comment]
🎉

**Pull request merged**